packages/core/test/lib/integrations/metadata.test.ts

```diff
@@ -63,4 +63,67 @@ describe('ModuleMetadata integration', () => {
 
     captureException(new Error('Some error'));
   });
+
+  test('Drops event if no stack frames have matching metadata', done => {
+    expect.assertions(0);
+
+    const options = getDefaultTestClientOptions({
+      dsn: 'https://username@domain/123',
+      enableSend: true,
+      stackParser,
+      integrations: [
+        moduleMetadataIntegration({ dropEvent: { ifNoStackFrameMetadataMatches: m => m?.team === 'backend' } }),
```

Comment:
```suggestion
        moduleMetadataIntegration({ dropEvent: { ifNoStackFrameMetadataMatches: m => m.team === 'backend' } }),
```

or can `m` be undefined here?

Reply:
I've changed the types to reflect that the parameter can be undefined. This allows users to filter on whether the metadata for a frame is defined!